Templates/BlobTrigger/index.js

```diff
@@ -1,4 +1,4 @@
 ﻿module.exports = function (workItem, context) {
-    console.log('Node.js blob trigger function processed work item ' + workItem.id);
+    context.log('Node.js blob trigger function processed work item', workItem.id);
```

Comment:
Not needed. Any params not matching a % placeholder are just concatenated on with spaces separating :)